pkg/config/config.go

```diff
@@ -329,6 +330,18 @@ func createModuleInfo(
 	return modInfo
 }
 
+// addKindToModules sets the kind to 'terraform' when empty.
+func (dc *DeploymentConfig) addKindToModules() {
```

Comment:
Can you add a test in `pkg/config/config_test.go` for this function. I think just something simple that passes in a `dc` that has empty string and verify that it comes out with terraform for kind. 

If you wanted to get fancier you could also test that `terraform` and `packer` do not change when defined explicitly, but I would settle for just the case above.

Reply:
good point. PTAL